src/tests/ftest/pool/evict_test.py

```diff
@@ -106,14 +107,14 @@ def evict_badparam(self, test_param):
         else:
             self.fail("Invalid yaml parameters - check \"params\" values")
         try:
-            # call daos evict api directly
-            self.pool.pool.evict()
+            # call dmg pool_evict directly
+            self.pool.dmg.pool_evict(pool=self.pool.pool.get_uuid_str())
         # exception is expected
-        except DaosApiError as result:
+        except CommandFailure as result:
             if test_param == "BAD_SERVER_NAME":
                 err = "-1003"
             else:
-                err = "-1005"
+                err = "1"
```

Comment:
We are mixing DAOS error codes and errno codes. Is it expected?

Reply:
I didn't have the opportunity to test the 'BAD_SERVER_NAME' scenario, only the bad UUID, that is why I only modified the this expected error.
I'm currently checking the dmg code under "/src/control/cmd/dmg/" to see if there are other return codes managed by the DMG command.

Reply:
While checking this with Jonathan (@berserk-fury) found that the function "exitWithError" in the /src/control/cmd/dmg/main.go L186 DMG always returns 1.
@mjmac is this correct? Are we missing some other place where different error codes are returned by DMG?

Reply:
dmg only returns 0 (success) or 1 (error). If you need to get at the DAOS error code you need to use the `-j,--json` flag and look at the status field. This is how the dmg helper on the C side works: https://github.com/daos-stack/daos/blob/master/src/common/tests_dmg_helpers.c#L233

Reply:
Thank you!

Reply:
Since we are moving to 'dmg' and this command responds back with rc=0 or rc=1, I'm updating the evict_test.py to be consistent with that behavior and see if command succeeds or fails with correct or bad parameters.

Reply:
@berserk-fury, this are the results in Functional after the changes:

`[2020-12-17T06:48:02.366Z] Running: avocado run --ignore-missing-references on --html-job-result on --tap-job-result off --filter-by-tags=pr,-hw --mux-yaml /tmp/tmpo1HCzn/pool-evict_test.yaml -- ./pool/evict_test.py
[2020-12-17T06:48:02.624Z] JOB ID     : 727ae6ad2d0dc8326f90cc34cd0af1166534b821
[2020-12-17T06:48:02.624Z] JOB LOG    : /var/tmp/ftest/avocado/job-results/job-2020-12-17T06.48-727ae6a/job.log
[2020-12-17T06:48:34.706Z]  (1/3) ./pool/evict_test.py:EvictTests.test_evict;badparams-container-hosts-pool-server_config-120c:  PASS (31.43 s)
[2020-12-17T06:48:34.706Z]  (2/3) ./pool/evict_test.py:EvictTests.test_evict_bad_server_name;badparams-container-hosts-pool-server_config-120c:  SKIP
[2020-12-17T06:48:56.635Z]  (3/3) ./pool/evict_test.py:EvictTests.test_evict_bad_uuid;badparams-container-hosts-pool-server_config-120c:  PASS (21.00 s)
[2020-12-17T06:48:56.636Z] RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0
[2020-12-17T06:48:56.636Z] JOB TIME   : 53.09 s
[2020-12-17T06:48:56.636Z] JOB HTML   : /var/tmp/ftest/avocado/job-results/job-2020-12-17T06.48-727ae6a/results.html
[2020-12-17T06:48:56.636Z] Total test time: 53s`